spec/parameters/query/offset.yml

```diff
@@ -1,5 +1,8 @@
 name: offset
 in: query
-description: Entry id at which to start pagination.
+description: |
+  Entry id at which to start pagination.
```

Comment:
hmm... didn't we decide that:

- we would split between `field` and `direction`?
- we would just have `direction` for now, because it internally defaults to `id` until we review what fields can be used to sort?

Reply:
1)

*  ``order``, which can be ascending or descending.
* ``field`` -> ``orderBy``, which is by default id if the endpoint does not define this filter, assuming that new transactions have a greater id.

2) I "believe" we ended deciding to keep the ``offset`` in case our users want to start paginating from a specific transaction. I have quoted the word because I cannot access older Slack messages to confirm it.

## Rationale

Imagine that a user is processing a set of transactions involving his/her account.

By having the offset, our user could decide to start pagination at a certain transaction and forget about the transactions that already processed.

What's more important, an offset would allow _set the first transaction to start pagination_. Since the transaction collection is continuously increasing, the offset ensures that the results will not be skipping transactions or showing them twice when advancing pages.

## Example

Assuming the transaction collection has the following entries: A,B,C,D, E , and pageSize is set to 2.

**Pagination *without* offset**

Query page 1 --> E, D
Query page 2 -->  B, C
Transaction F is added to the collection
Query page 2 -->  C, D

⚠️ Note that D is returned twice because E was added between queries. ⚠️ 

**Pagination *with* offset**

Assuming offset = E
Query page 1 --> C,D
Query page 2 -->  A,B
Transaction F is added
Query page 2 -->  A,B

In this case, the pagination works as expected, even the collection receives new transactions in between.

Reply:
I'm not saying we didn't decide that we'd have the offset, but didn't we decide we'd implement it in a near future?